ee/clickhouse/models/person.py

```diff
@@ -20,9 +20,10 @@
 from ee.kafka_client.topics import KAFKA_PERSON, KAFKA_PERSON_UNIQUE_ID
 from posthog.models.person import Person, PersonDistinctId
 from posthog.models.utils import UUIDT
+from posthog.settings import TEST
 from posthog.utils import is_clickhouse_enabled
 
-if is_clickhouse_enabled():
+if is_clickhouse_enabled() and TEST:
 
     @receiver(post_save, sender=Person)
     def person_created(sender, instance: Person, created, **kwargs):
```

Comment:
non-blocking just curious

The original ticket is about deletion, what's the reason to special case `person_created` and `person_distinct_id_created` as well?

Reply:
These were used more pre-plugin server in ingestion. Nowadays only relics go through these code paths - split_person was the last one that hadn't been updated.